x/simulation/params.go

```diff
@@ -142,3 +145,23 @@ func (w WeightedProposalContent) DefaultWeight() int {
 func (w WeightedProposalContent) ContentSimulatorFn() simulation.ContentSimulatorFn {
 	return w.contentSimulatorFn
 }
+
+//-----------------------------------------------------------------------------
+// Param change proposals
+
+// RandomParams returns random simulation consensus parameters
+func RandomConsensusParams(r *rand.Rand) *abci.ConsensusParams {
+	return &abci.ConsensusParams{
+		Block: &abci.BlockParams{
+			MaxBytes: int64(simulation.RandIntBetween(r, 20000000, 30000000)),
+			MaxGas:   -1,
+		},
+		Evidence: &abci.EvidenceParams{
+			MaxAgeNumBlocks: int64(simulation.RandIntBetween(r, 100000, 500000)),
```

Comment:
Mentioned this in the original issue, these two values needed to be based on the unbonding period defined in x/staking. I think you need to use `appState`, decode x/staking genesis state, and get the unbonding period.